tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css

```diff
@@ -3,235 +3,212 @@ Created, maintained & Owned by: Tauqeer Khan
 Date: 16/07/21
 ******************************************/
 
-body{
-    font-family: arial;
-    margin: 10px auto;
-    background-color: lightgray;
-    
+body {
+  font-family: arial;
+  margin: 10px auto;
+  background-color: lightgray;
```

Comment:
you can just go with `background` here totally up to you

Reply:
yep, I will consider these changes, these projects were coded when we were learning CSS and were not good at it. @coder-mezab